osm2gtfs/creators/ci_abidjan/trips_creator_ci_abidjan.py

```diff
@@ -0,0 +1,209 @@
+# coding=utf-8
+
+import collections
+from datetime import timedelta, datetime
+import transporthours
+from transitfeed import ServicePeriod
+
+from osm2gtfs.creators.trips_creator import TripsCreator
+from osm2gtfs.core.helper import Helper
+from osm2gtfs.core.elements import Line
+
+
+class TripsCreatorCiAbidjan(TripsCreator):
+
+    _DAYS_OF_WEEK = [ 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday' ]
+    _DEFAULT_START_TIME = "05:00:00"
+    _DEFAULT_END_TIME = "22:00:00"
+    _DAY_ABBREVIATIONS = {
+            'monday': 'Mo',
+            'tuesday': 'Tu',
+            'wednesday': 'We',
+            'thursday': 'Th',
+            'friday': 'Fr',
+            'saturday': 'Sa',
+            'sunday': 'Su'
+        }
+    _DEFAULT_HEADWAY = 30 # in minutes
+
+    def _date_range(self, start, end):
```

Comment:
as this function is only used in `_days_abbreviation_from_transport_hour`, I suggest to put it inside that function

Reply:
ok